### DISCUSSION TITLE
How to disable indentline in NvimTree

### DISCUSSION THREAD

**Discussion by Song-Tianxiang** [nvim-mini/mini.nvim#762] [Q&A] [answered]
As title, how can I disable mini.indentline in nvimtree?

**Comment by echasnovski** [comment, marked as answer]
Take a look at [this part of 'mini.indentscope'](https://github.com/echasnovski/mini.nvim/blob/a683bfe8e03293e3bb079e24c94e51977756a3ec/doc/mini-indentscope.txt#L65-L71) documentation and [general disabling recipes](https://github.com/echasnovski/mini.nvim/blob/a683bfe8e03293e3bb079e24c94e51977756a3ec/doc/mini.txt#L292).

**Reply by Song-Tianxiang** [reply]
Thanks for your reply and all your great plugins. Mini really wonderful.
